tests/test_api_endpoints.py

```diff
@@ -36,6 +36,13 @@ def create_new_session():
     })
     return session
 
+def verify_database_is_clean():
```

Comment:
Nice! I have almost the same maybe let's agree on what we're merging into main

Reply:
cool man <3